lib/lambda/sinkMainProcessors.ts

```diff
@@ -94,6 +117,29 @@ const getOneMacRecordWithAllProperties = (
     console.log(`event after transformation: ${JSON.stringify(oneMacRecord, null, 2)}`);
 
     return oneMacRecord;
+  } else if (isRecordALegacyOneMacRecord(record, kafkaSource)) {
```

Comment:
Same here about the `else/if`

Reply:
I understand its the same functionally however i think having multiple if blocks not connected by an else is less "readable" than someone having to determine if the prior if block terminates with a return.